src/bridges/bridge_argon2id_reference.c

```diff
@@ -18,7 +18,7 @@
 #include "core.c"
 #include "blake2/blake2b.c"
 
-#if defined (__riscv)
+#if defined (__riscv) || defined(__loongarch__) || defined(__powerpc__) || defined(__mips__)
```

Comment:
So is this -- we should selectively enable the optimized implementation for the ported architectures, and leave the rest as fallback.